src/tools/exports/liste-cnfs-pour-carto-v2.js

```diff
@@ -0,0 +1,402 @@
+#!/usr/bin/env node
+'use strict';
+const path = require('path');
+const cli = require('commander');
+const fs = require('fs');
+const CSVToJSON = require('csvtojson');
+const { ObjectId } = require('mongodb');
+const { formatOpeningHours } = require('../../services/conseillers/common');
+const { execute } = require('../utils');
+
+const departements = require('./departements-region.json');
+const deps = new Map();
+
+for (const value of departements) {
+  deps.set(String(value.num_dep), value);
+}
+
+const readCSV = async filePath => {
+  try {
+    // eslint-disable-next-line new-cap
+    const structuresIds = await CSVToJSON({ delimiter: 'auto' }).fromFile(filePath);
+    return structuresIds;
+  } catch (err) {
+    throw err;
+  }
+};
+
+const codePostal2departementRegion = cp => {
+  if (!/^.{5}$/.test(cp)) {
+    return null;
+  }
+  let dep;
+  if ((dep = cp.match(/^9[78]\d/))) {
+    // DOM
+    return deps.get(dep[0]);
+  } else if ((dep = cp.match(/^20\d/))) {
+    if (['200', '201'].includes(dep[0])) {
+      // Corse du sud
+      return deps.get('2A');
+    }
+    if (['202', '206'].includes(dep[0])) {
+      // Haute Corse
+      return deps.get('2B');
+    }
+  } else if ((dep = cp.match(/^\d\d/))) {
+    // Le reste
+    return deps.get(String(dep[0]));
+  }
+  return null;
+};
+
+function cleanPhoneNumber(number) {
+  let cleaned = number.replace(/\D/g, '');
+
+  if (cleaned.startsWith('33') && cleaned.length === 11) {
+    cleaned = '0' + cleaned.substring(2);
+  } else if (cleaned.length === 12) { // Pour les DOM comme +590590478535
+    cleaned = '0' + cleaned.substring(3);
+  }
+
+  return `${cleaned.substring(0, 2)}${cleaned.substring(2, 4)}${cleaned.substring(4, 6)}${cleaned.substring(6, 8)}${cleaned.substring(8, 10)}`;
+}
+
+cli.description('Export liste CNFS pour carto txt')
+.option('-c, --csv <path>', 'Chemin fichier CSV')
+.helpOption('-e', 'HELP command')
+.parse(process.argv);
+
+const uniformiseAdresse = a =>
+  a?.replace(/avenue/gi, 'av.')
```

Comment:
```suggestion
  a.replace(/avenue/gi, 'av.')
```

vu qu'il y en a pas en dessous mais ca changera rien